features/daily/asterisk.feature

```diff
@@ -1,7 +1,7 @@
 Feature: Asterisk
 
     Scenario: Asterisk is correctly installed
-        Then Asterisk may open at most 8192 file descriptors
+        Then Asterisk may open at most "8192" file descriptors
```

Comment:
The double quote is only here to allow spaces within the variable value. They are not strongly necessary. Do we have a convention to surround every variable with quotes? I don't have a strong opinion on this:
* no quotes is more natural to read _in this case_
* quotes give the hint "this is a variable"

What's your opinion on this?

In any case, we should write down this kind of convention in a DEVELOP.md file.

Reply:
@sduthil we added a style guide in the README yesterday. @fblackburn1 added the variable quoting today.

Reply:
Thanks :)